src/main/java/com/cloudbees/jenkins/support/impl/AboutJenkins.java

```diff
@@ -739,7 +739,7 @@ private String getLabelString(Node n) {
                     }
                 }
                 for (String resourcePath : Arrays.asList(
-                        "/WEB-INF/slave.jar", // note that as of 2.33 this will not be present (anyway it is the same as war/WEB-INF/lib/remoting-*.jar, printed above)
+                        "/WEB-INF/agent.jar", // note that as of 2.33 this will not be present (anyway it is the same as war/WEB-INF/lib/remoting-*.jar, printed above)
```

Comment:
Not sure what we are talking about here @Dohbedoh 
This is/was a jar within jenkins WAR ?
Maybe we can just remove the line if the minimum version of jenkins the plugin support do not find any of them?

Reply:
yeah we can remove this and `remoting.jar` I think: https://github.com/jenkinsci/jenkins/pull/2633/files